ruby_basics/1_data_types/exercises/numbers_exercises.rb

```diff
@@ -0,0 +1,35 @@
+def addition_exercise(a, b)
+  ## write a function that adds two numbers
```

Comment:
You only need one pound symbol for ruby comments: 
`# write a function that adds two numbers`

same for the rest of the comments in this file 😄

Reply:
"Write a function" could perhaps be a confusing and lead to students writing a method within the test method.

The format you used in the strings file is perfect imo, can we use the same format in this file, for example: "return the result of a and b added together"